src/Core/Services/Implementations/MultiServicePushNotificationService.cs

```diff
@@ -16,44 +13,10 @@ public class MultiServicePushNotificationService : IPushNotificationService
     private readonly ILogger<MultiServicePushNotificationService> _logger;
 
     public MultiServicePushNotificationService(
```

Comment:
I am a bit confused as to what's all going on here and why this is necessary,

Reply:
This service generating its own services to multicast out steps on the toes of DI, so all I'm doing is removing all that service initialization logic and putting it in our [`ServiceCollectionExtensions`](https://github.com/bitwarden/server/pull/4450/files/c53037ea1c3ee034e4a586fad6cbed718118aae3#diff-083f45daf5a897e9017dc669a0def5c46306a936409956632026a7cce2cee933R296-R317)